### DISCUSSION TITLE
Bleachbit Distribution - Packaged into Installer?

### DISCUSSION THREAD

**Discussion by Swatto86** [bleachbit/bleachbit#1327] [Q&A] [answered]
Hello all, and happy new year!

How is Bleachbit packaged into an installer please? I’m looking at the best way to distribute my own Python applications.

Thanks very much

**Comment by az0** [comment, marked as answer]
Check this code https://github.com/bleachbit/bleachbit/blob/master/windows/setup_py2exe.py

For a while it was an issue to support new versions of Python3 with py2exe, but recently the situation improved. Their repository shows signs of life, and on my repository, Max has a new PR for this.
